DependencyInjection/Configuration.php

```diff
@@ -39,6 +39,8 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('domain_message_prefix')->defaultValue('domain.')->end()
                 ->scalarNode('domain_message_serializer')->cannotBeEmpty()->end()
+                ->scalarNode('resend_producer')->cannotBeEmpty()->end()
```

Comment:
These should not be required for several reasons.  Projects that use this bundle may not need this functionality and it will break compatability with older projects when they decide to upgrade.